### PR TITLE
UI updates and Upload Feedback link (SCP-4897, SCP-5018, SCP-5017)

### PR DESCRIPTION
**Pull request by ehanna4**
Fixes addressed in this PR:
- The loading spinner is more appropriately sized 
- On the Clustering tab you can now add multiple clustering forms
- Indications in both the Expression and Clustering steps and also the Nav Panel that you will be uploading the AnnData file at the end in the AnnData tab
- Add a button to toggle between AnnData and Classic mode if the user has not yet uploaded any file 

I plan to merge this after Jon's PR merges so will address conflicts here as needed.

To Test: 

NOTE - the changes here are almost entirely visual don't bother actually trying to upload here

- Pull this branch
- Start up local and navigate to a study that has no files yet
- When you click to the upload page observe the spinner 
- When you land on the option page observe the link to the survey
- Choose AnnData and see in the Navpanel at the bottom the link to the survey
- Hover over the "?" to read the text about uploading a file at the end
- Check out the Clustering tab and the Expression tab to see the text about uploading a file at the end
- In the Clustering tab observe that there is now a button "Add clustering" allowing for adding another form.

Improved spinner:

https://user-images.githubusercontent.com/54322292/225704711-8ec26e6f-979f-4747-a73c-0dddd28140e7.mov

On the option page the link to the survey
Screen Shot 2023-03-10 at 12 23 48 PM

The explanations in Clustering and Expression Matrices tabs that you upload at the end
Screenshot 2023-03-13 at 1 54 43 PM
Screenshot 2023-03-13 at 1 54 49 PM

Explanation about where to upload in the NavPAnel
Screenshot 2023-03-13 at 2 14 31 PM

### CODE REVIEW

File: app/javascript/components/upload/UploadWizard.jsx [hunk 1]
```diff
@@ -465,22 +469,34 @@ export function RawUploadWizard({ studyAccession, name }) {
     }
   }
 
+  /** return a button for switching to the other experience (AnnData or Classic) */
+  function getOtherChoiceButton() {
+    const otherOption = isAnnDataExperience ? 'classic upload' : 'AnnData upload'
+    return <button
+      className="btn terra-primary-btn margin-left-30"
```
Comment (eweitz) [inline app/javascript/components/upload/UploadWizard.jsx:476]: Blocker: this button looks off in the UI.  I suspect you meant `.terra-secondary-btn`, as it seems like a better fit than `.primary-btn` and `.terra-primary-btn` is not defined.

P.S.: I hadn't appreciated this, but writing out those selectors highlights how their naming patterns are inconsistent in multiple ways!  Fixing that is out of scope here, of course; I just found it noteworthy.

File: app/javascript/components/upload/UploadExperienceSplitter.jsx [hunk 2]
```diff
@@ -18,6 +19,9 @@ function RawUploadExperienceSplitter({
         <div className="splitter-info-outside ">
           <div className='split-title'>
           Choose how to upload expression, conventional metadata, and cluster data
+            { <div className='font-size-14'><a href="https://forms.gle/dRUVSh7WAz9Dh6Ag8" target="_blank" title="Take a brief survey on AnnData data upload">
+              Tell us about your experience with data upload <FontAwesomeIcon icon={faLink}/>
```
Comment (eweitz) [inline app/javascript/components/upload/UploadExperienceSplitter.jsx:23]: [Most major products omit icons in external and/or new tab links](https://github.com/broadinstitute/single_cell_portal_core/pull/1716#discussion_r1096296306), and SCP almost always does as well.  Tweaking the copy a bit might also be worthwhile clarity.

```suggestion
              Tell us about your experience with AnnData upload!
```

**Review by eweitz — APPROVED** [review]
Code and functionality look good overall!  I manually tested locally.

I noted 3 trivial blockers, 2 user-facing and 1 in maintainability.  Beyond that I note a handful of non-blocking suggestions for UI and code style.

I recall we'd discussed [adding a Mixpanel property](https://docs.google.com/presentation/d/1QzHy48am_lqk8T3rFSL-4OmIRyaTTK_B9ozZC6HjEVs/edit#slide=id.g216c356411d_0_5) for whether you’re in AnnData or classic mode.  It strikes me as a nice to have, but not worth holding things up here.  Is it worth a ticket, or maybe just something we keep in mind for later until there's a clearer use case?

Finally, while testing this locally, I notice we write "AnnData files, typically formatted with the file extension .h5ad. [See reference documentation](https://anndata.readthedocs.io/)."  Can we change that to "AnnData files are typically..."?  The other tabs speak in sentences, so this seems like it'd be a bit more consistent and polished.

**Review by michelle-vel — APPROVED** [review]
These changes look really great! I'll be interested to see the metrics in how many people select the AnnData (beta) upload button. Is there a way to track it that we have already set up?

**Comment by ehanna4** [comment]
Hey @michelle-vel thanks for the review! Leyla has set up a board in mixpanel here: https://mixpanel.com/s/24eB20 that includes a report on users choosing the AnnData upload option. Currently there's no data since it's not enabled on Prod but will be good to have that link here for easy reference so thanks for asking that question!

**Review by jlchang — APPROVED** [review]
Functional review, observed all described changes. Looks good.
